Snippets/print-json.py

```diff
@@ -0,0 +1,153 @@
+import fontTools.ttLib as ttLib
+from fontTools.ttLib.ttVisitor import TTVisitor
+from fontTools.misc.textTools import Tag
+from array import array
+
+
+class JsonVisitor(TTVisitor):
```

Comment:
ah, this is fun. 
curious, why didn't you build a raw dict and pass it on to json library to serialize and opted for printing strings directly? For the sake of this snippet either is ok.

Reply:
Right. That didn't occur to me. This was more of an inspector fun piece of code for debugging purposes kind of thing.